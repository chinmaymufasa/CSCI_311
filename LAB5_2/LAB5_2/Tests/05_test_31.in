2
193
2
67
2
23
2
178
2
75
2
105
2
85
2
21
2
46
2
82
2
15
2
192
2
115
2
39
2
188
2
135
2
136
2
105
2
19
2
122
4
5
6
1
122
-1
