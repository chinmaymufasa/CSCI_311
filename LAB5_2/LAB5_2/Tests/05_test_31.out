75 23 19 15 21 46 39 67 115 85 82 105 105 178 135 122 136 192 188 193 
15 19 21 23 39 46 67 75 82 85 105 105 115 122 135 136 178 188 192 193 
15 21 19 39 67 46 23 82 105 105 85 122 136 135 188 193 192 178 115 75 
122
